src/modules/task/internal/store/task-db.ts

```diff
@@ -0,0 +1,27 @@
+import { Schema, Types } from 'mongoose';
+
+export interface TaskDB {
+  _id: Types.ObjectId;
+  account: Types.ObjectId;
+  active: boolean;
+  name: string;
+}
+
+export const taskDbSchema: Schema = new Schema<TaskDB>(
+  {
+    active: { type: Boolean, required: false, default: true },
```

Comment:
Should this be required?

Reply:
whenever we create a task by default it should be active, when we delete a task , it does not delete that record just update the active flag to false.

Reply:
Yeah, we always need it, so `required` should be `true`?

Reply:
I think we should be able to create an active task even if we just provide the name of the task, so I think this should not be required as a must field .